modules/migrations/github.go

```diff
@@ -97,15 +98,31 @@ func NewGithubDownloaderV3(ctx context.Context, baseURL, userName, password, tok
 			},
 		},
 	}
+
 	if token != "" {
-		ts := oauth2.StaticTokenSource(
-			&oauth2.Token{AccessToken: token},
-		)
-		client = oauth2.NewClient(downloader.ctx, ts)
-	}
-	downloader.client = github.NewClient(client)
-	if baseURL != "https://github.com" {
-		downloader.client, _ = github.NewEnterpriseClient(baseURL, baseURL, client)
+		tokens := strings.Split(token, ",")
+		for _, token := range tokens {
+			token = strings.TrimSpace(token)
+
+			ts := oauth2.StaticTokenSource(
+				&oauth2.Token{AccessToken: token},
+			)
+			client = oauth2.NewClient(downloader.ctx, ts)
+
+			githubClient := github.NewClient(client)
+			if baseURL != "https://github.com" {
+				githubClient, _ = github.NewEnterpriseClient(baseURL, baseURL, client)
+			}
+			downloader.clients = append(downloader.clients, githubClient)
+			downloader.rates = append(downloader.rates, nil)
+		}
+	} else {
+		githubClient := github.NewClient(client)
+		if baseURL != "https://github.com" {
+			githubClient, _ = github.NewEnterpriseClient(baseURL, baseURL, client)
+		}
+		downloader.clients = append(downloader.clients, githubClient)
+		downloader.rates = append(downloader.rates, nil)
```

Comment:
As both `112-117` and `120-125` do exactly the same thing, I would prefer outsourcing this into its own method with three parameters: the `client`, the `baseURL` and the `downloader`.

Reply:
The frontend will reuse `token` input box, but allow multiple tokens with a comma splited string.

Reply:
But the front-end currently does not notify you about being able to do that.
At least you should add a description notifying the user that they can pass multiple access tokens comma-separated to speed up the repo migration.

Reply:
By the way, are the other migrations also slowed down by rate limits?
Because if that is the case I would prefer making this a more general solution as the other migrations will then need the same solution.

Reply:
As I know, currently only Github has the rate limit. And Gitea and Gogs has no rate limit currently. I don't know GitLab and OneDev.

Reply:
OneDev has no rate limit (at the time of my PR).

Reply:
@delvh  Done.